peps/pep-0224.rst

```diff
@@ -3,11 +3,11 @@ Title: Attribute Docstrings
 Author: Marc-André Lemburg <mal@lemburg.com>
 Status: Rejected
 Type: Standards Track
-Content-Type: text/x-rst
 Created: 23-Aug-2000
 Python-Version: 2.1
 Post-History:
 
+.. rejected::
```

Comment:
This PEP contains a part near the end discussion the rejection, its just rather long and buried within another section. If you added a ref target label below (e.g. `224-rejection`), you could link to the `Comments from our BDFL` section below describing the rejection reasons, e.g.

```rst
.. rejected::
   This PEP has been rejected.
   See :ref:`224-rejection` for more information.
```